packages/devtools_shared/lib/src/server/server_api.dart

```diff
@@ -60,6 +60,7 @@ class ServerApi {
     ServerApi? api,
     DtdInfo? dtd,
   }) {
+    print('Handling ${request.url.path}');
```

Comment:
delete this print

Reply:
oops :) done

Consider enabling https://dart.dev/tools/linter-rules/avoid_print . Then the analyzer would catch this.